hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -302,7 +302,7 @@ public static Map<String, FileStatus[]> getFilesInPartitions(HoodieEngineContext
                                                                HoodieMetadataConfig metadataConfig,
                                                                String basePathStr,
                                                                String[] partitionPaths) {
-    try (HoodieTableMetadata tableMetadata = HoodieTableMetadata.create(engineContext, metadataConfig, basePathStr, true)) {
+    try (HoodieTableMetadata tableMetadata = HoodieTableMetadata.create(engineContext, metadataConfig, basePathStr)) {
```

Comment:
this is used only in flink and repair tool for spark. I will let @danny0405 certify the flow for flink.